lamindb/curators/core.py

```diff
@@ -264,6 +264,12 @@ def __repr__(self) -> str:
 
         cls_name = colors.green(self.__class__.__name__)
 
+        # Hide AtomicDataFrameCurator implementation detail
```

Comment:
Why do we want to hide this important information?

Reply:
Users don't know what an `AtomicDataFrameCurator` is. We're hiding it completely and it's undocumented. I thought that it's more confusing than helpful.

Reply:
By that logic, shouldn't you hide any internal/technical class? I don't think this is advisable. I think we should not mess with this so that people have an easy time debugging and guessing the structure of the code. I think `AtomicDataFrameCurator` should be as visible as `SlotsCurator`.

If we want a simpler name, one could also call it `SimpleDataFrameCurator`. ☺️

Reply:
>  it's undocumented

It should feature under `lamindb.curators.core`, I think, but not under `lamindb.curators`